server/src/main/java/org/candlepin/guice/CandlepinContextListener.java

```diff
@@ -224,6 +229,20 @@ private void initializeSubsystems(Injector injector) throws Exception {
             capabilities.add(CandlepinCapabilities.KEYCLOAK_AUTH_CAPBILITY);
         }
 
+        // Init CRL file
+        this.crlFileUtil = injector.getInstance(CrlFileUtil.class);
+        String filePath = getCrlFilePath();
+        File crlFile = new File(filePath);
+        if (!crlFile.exists() || crlFile.length() == 0) {
+            try {
+                this.crlFileUtil.initializeCRLFile(crlFile, Collections.emptyList());
+            }
+            catch (IOException e) {
+                e.printStackTrace();
```

Comment:
We should write these to the log and probably rethrow the exception as well.

Reply:
Done.